gps.cpp

```diff
@@ -287,11 +288,14 @@ void GPS::processGPRMC(const QString & line) {
     QString decLat = convertToDecimal(latitude, latDirection);
     QString decLon = convertToDecimal(longitude, lonDirection);
 
-    if ((m_dashboard->gpsFIXtype() == "GPS only") ||(m_dashboard->gpsFIXtype() == "DGPS") )
+    if ((m_dashboard->gpsFIXtype() == "GPS only") || (m_dashboard->gpsFIXtype() == "DGPS") )
     {
     m_dashboard->setgpsLatitude(decLat.toDouble());
     m_dashboard->setgpsLongitude(decLon.toDouble());
+    if ((hdop >= 50) || (speed >= 15))  // This avoids that the GPS speed fluctuates when standing and hdop is low
```

Comment:
The hdop threshold here might be a bit high?

Reply:
Which threshold do you think would be better ?